### PR TITLE
[luv-363] fix: dynamic CLI list in /policies subheading

### PR DESCRIPTION
**Pull request by NiveditJain**
## Summary
- Dashboard `/policies` activity-tab subheading used to read **"Policy evaluations for Claude · enabled policies 37/43"** — a leftover from when failproofai only supported Claude Code.
- Now reads a dynamic list of installed CLIs pulled from the existing `getHooksConfigAction()` payload: `"across Claude Code, Cursor"` when 1–3 are installed, `"across N agents"` when 4+, and just `"Policy evaluations"` when none are installed.
- No new server work; no API/payload changes.
- CHANGELOG.md entry sits under the existing `## 0.0.11-beta.0 — 2026-05-13` section (the cut date of the current version per the repo's `chore: bump version` automation), Fixes subsection.

## Commits
1. **fix**: replace hardcoded "for Claude" in `/policies` subheading + Fix entry in CHANGELOG.
2. **chore**: (now reverted in commit 3 per CodeRabbit) — initially split the CHANGELOG into a same-version-different-date heading.
3. **fix**: collapse CHANGELOG back into the single `0.0.11-beta.0 — 2026-05-13` section per CodeRabbit feedback — the repo's CHANGELOG convention is one section per version (dated to the cut), not per-PR.

## Test plan
- [x] `bunx tsc --noEmit` clean
- [x] `bun run lint` clean (only the pre-existing unrelated `` warning)
- [x] `bun run test:run` → 1611/1611 passing
- [ ] Visual check: open `/policies` activity tab locally and confirm the subheading reflects the installed CLI set
- [ ] Edge case: uninstall all CLIs → subheading falls back to "Policy evaluations"
- [ ] Edge case: install 4+ CLIs → subheading collapses to "across N agents"

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Bug Fixes**
  * Policies activity tab now shows a dynamic installed-CLI integrations heading: lists up to three labels, collapses to “across N agents” when 4+ integrations are present, and falls back to “Policy evaluations” when none are installed.

[Review Change Stack](https://app.coderabbit.ai/change-stack/exospherehost/failproofai/pull/358?utm_source=github_walkthrough&utm_medium=github&utm_campaign=change_stack)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.0.11-beta.0 — 2026-05-18
+
+### Fixes
+- Dashboard `/policies` activity-tab subheading: replace the hardcoded "Policy evaluations for Claude" with a dynamic list of installed CLIs ("Policy evaluations across Claude Code, Cursor"), collapsing to "across N agents" when 4 or more are installed and falling back to "Policy evaluations" when none are. Reads from the existing `getHooksConfigAction()` payload — no new server work. The text was a leftover from when failproofai only supported Claude Code and was inaccurate against the now-7-CLI surface (Claude, Codex, Copilot, Cursor, OpenCode, Pi, Gemini) (#358).
+
```
[the remark this answers is not in the document]
Comment (NiveditJain) [reply CHANGELOG.md]: Addressed in 010046d — collapsed back to the single `## 0.0.11-beta.0 — 2026-05-13` section per your recommendation (option 1). The repo convention is one section per version, dated to the cut (the `chore: bump version` automation owns the date); subsequent PRs add entries under the existing section regardless of when they merge. The temporary 2026-05-18 split came from following a too-literal reading of the failproofai release-prep hook's wording — flagged separately for the policy author.
